pygame/math.py

```diff
@@ -0,0 +1,1529 @@
+""" Math module """
+
```

Comment:
Sure.  Whose names do I include? 
For math_test.py, most of it came from the original pygame repo, so leave it as is?